### PR TITLE
[video.js] Add responsive to VideoJsPlayerOptions & VideoJsPlayer

### PR DESCRIPTION
**Pull request by blzaugg**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
  - https://docs.videojs.com/player#responsive
  - https://docs.videojs.com/tutorial-options.html#responsive

### CODE REVIEW

File: types/video.js/video.js-tests.ts [hunk 1]
```diff
@@ -148,6 +149,10 @@ videojs('example_video_1', playerOptions).ready(function() {
 
     const networkState: videojs.NetworkState = this.networkState();
 
+    const responsive: boolean = this.responsive();
+
+    this.responsive(false);
+
```
Comment (blzaugg) [inline types/video.js/video.js-tests.ts:155]: Member test added

**Comment by blzaugg** [comment]
@joeflateau, @gjanblaszczyk, @sroucheray any chance of taking a look at this?

**Review by joeflateau — CHANGES_REQUESTED** [review]
please revert all of the quotes to their previous form (single). looks good beyond that

**Review by joeflateau — APPROVED** [review]
looks good

**Comment by blzaugg** [comment]
Ready to merge

**Pull request merged**